oshc/main/templates/base.html

```diff
@@ -1,49 +1,40 @@
+	{% include 'header.html' %}
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	{% if site.title %}<title>{% if title %}{{ title }} &ndash; {% endif %}{{ site.title }}{% if not title and site.subtitle %} &ndash; {{ site.subtitle }}{% endif %}</title>{% endif %}
 </head>
-
 <body>
-
-  <div class="container">
-    <nav class="navbar navbar-default">
-      <div class="container-fluid">
-        <div class="navbar-header">
-           <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-            </button>
-            <a class="navbar-brand" href="/">{{ site.title }}</a>
-        </div>
-        <div class="navbar-collapse collapse">
-          <ul class="nav navbar-nav">
-            <li><a href="/sessions/">Sessions</a></li>
-            <li><a href="https://github.com/OpenSourceHelpCommunity">Resources</a></li>
-            <li><a href="mailto:opensourcehelpcommunity@gmail.com">Contact</a></li>
-            <li><a href="https://opensourcehelp.herokuapp.com/">Join Us!</a></li>
-          </ul>
-        </div>
-      </div>
-    </nav>
-  </div>
-
-  <div id="content">
-    {% block content %}{% endblock %}
-  </div>
-<script
-  src="https://code.jquery.com/jquery-3.2.1.min.js"
-  integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-  crossorigin="anonymous"></script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-
-
+	<nav class="navbar navbar-inverse navbar-static-top">
+		<div class="container">
+			<div class="navbar-header">
+				<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+					<span class="sr-only">Toggle navigation</span>
+					<span class="icon-bar"></span>
+					<span class="icon-bar"></span>
+					<span class="icon-bar"></span>
+				</button>
+				<a class="navbar-brand" href="#">Open Source Help Community</a>
+			</div>
+			<div id="navbar" class="navbar-collapse collapse">
+				<ul class="nav navbar-nav navbar-right">
+					<li><a href="/sessions/">Sessions</a></li>
+					<li><a href="https://github.com/OpenSourceHelpCommunity">Resources</a></li>
+					<li><a href="mailto:opensourcehelpcommunity@gmail.com">Contact</a></li>
+					<li><a href="https://opensourcehelp.herokuapp.com/">Join Us!</a></li>
+				</ul>
+			</div>
+		</div>
+	</nav>
+
+	<div id="content">
+	{% block content %}{% endblock %}
+	</div>
+
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Built with Bootstrap so it's mostly responsive. Have to write `media-query` for some overflows. I was waiting for everyone's approval on the design. I am a little lazy :sweat_smile: